nixos/doc/manual/configuration/linux-kernel.xml

```diff
@@ -66,6 +66,58 @@ nixpkgs.config.packageOverrides = pkgs:
   sets the kernel’s TCP keepalive time to 120 seconds. To see the available
   parameters, run <command>sysctl -a</command>.
  </para>
+ <section>
+  <title>Configure your kernel</title>
```

Comment:
I would title this `Building a customized kernel` or something like that.

Reply:
is `Customize your kernel` ok ?